src/github/utils.ts

```diff
@@ -255,7 +255,7 @@ export function convertGraphQLEventType(text: string) {
 	}
 }
 
-export function parseGraphQLComment(comment: GraphQL.ReviewComment): IComment {
+export function parseGraphQLComment(comment: GraphQL.ReviewComment, isResolved: boolean): IComment {

```

Comment:
Why not update `GraphQL.ReviewComment` to include the `resolved` property so that you don't have to pass it as an additional param here?

Reply:
The property is actually at the thread level instead of on individual comments, but I thought adding a concept of threads instead of the comments we use everywhere was too big of a refactoring